k8s/k8s.go

```diff
@@ -300,7 +300,10 @@ func CmdAddK8s(ctx context.Context, args *skel.CmdArgs, conf types.NetConf, epID
 	endpoint.Spec.MAC = mac.String()
 	endpoint.Spec.InterfaceName = hostVethName
 	endpoint.Spec.ContainerID = epIDs.ContainerID
-	logger.WithField("endpoint", endpoint).Info("Added Mac, interface name, and active container ID to endpoint")
+	logger.WithFields(logrus.Fields{
+		"endpoint": endpoint,
+		"mac":      mac.String(),
```

Comment:
I don't think we need this change - the endpoint already has the mac set, so it gets logged out already.

Reply:
See line 300